### PR TITLE
Show duplicate status again

### PR DESCRIPTION
**Pull request by dominikl**
Should fix the "companion file" issue, see bug reports line 10:
https://docs.google.com/spreadsheets/d/1SDXHxHpWQQ3VBpsGAOXdpZ4H1JdSMTQZHEu3Eg1CAQk/edit#gid=1885911056

Luckily that was just a display issue.

/cc @mtbc

### CODE REVIEW

**Comment by dominikl** [comment]
The missing change from "Pending..." to "Cancelled..." (line 11) was basically the same issue, StatusLabel not getting notified when the Status has changed. Should be fixed now.

File: components/insight/SRC/org/openmicroscopy/shoola/env/data/util/StatusLabel.java [hunk 1]
```diff
@@ -67,6 +67,9 @@
     /** The text indicating the scanning steps. */
     public static final String SCANNING_TEXT = "Scanning...";
 
+    /** The text indicating the scanning steps. */
+    public static final String CANCELLED_TEXT = "Cancelled...";
```
Comment (mtbc) [inline components/insight/SRC/org/openmicroscopy/shoola/env/data/util/StatusLabel.java]: no ellipsis suffix, was simply "Cancelled"

**Review by mtbc — APPROVED** [review]
In local testing both issues are now fixed for me.

**Comment by mtbc** [comment]
Also worked fine with this morning's eel merge build against pub-omero.

**Comment by joshmoore** [comment]
Assuming @rgozim doesn't have any objections.

**Pull request merged**